packages/admin/package.json

```diff
@@ -31,11 +31,13 @@
     "next-logger": "3.0.1",
     "nookies": "^2.5.2",
     "number-to-words": "^1.2.4",
+    "pino": "^8.18.0",
     "react": "18.2.0",
     "react-device-detect": "^2.1.2",
     "react-dom": "18.2.0",
     "react-gtm-module": "^2.0.11",
-    "tinymce": "6.8.2"
+    "tinymce": "6.8.2",
+    "uuid": "9.0.1"
```

Comment:
just making sure to add these specifically to admin as well while I remember because otherwise weird stuff can happen with the `yarn workspace` setup - these will be consumed in a future PR